app/src/main/java/io/bankingofthings/iot/error/ActionNotActivatedError.kt

```diff
@@ -0,0 +1,3 @@
+package io.bankingofthings.iot.error
+
+class ActionNotActivatedError : Throwable()
```

Comment:
Please use `Exception`, not `Throwable`. 
The difference is that `Exception` is meand to be caught. 
`Throwable` is base class and should not be used at all